extensions/src/AWSSDK.Extensions.CloudFront.Signers/AWSSDK.Extensions.CloudFront.Signers.nuspec

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package> 
+  <metadata> 
+    <id>AWSSDK.Extensions.CloudFront.Signers</id>
+    <title>AWSSDK - Extensions for AWS CloudFront</title>
+    <version>4.0.1.0-preview</version>
+    <authors>Amazon Web Services</authors>
+		<description>
+			This package contains extension methods for creating signed URLs for Amazon CloudFront distributions and
+			for creating signed cookies for Amazon CloudFront distributions using canned or custom policies.
+		</description>
+    <language>en-US</language>
+    <license type="expression">Apache-2.0</license>
+    <projectUrl>https://github.com/aws/aws-sdk-net/</projectUrl>
+    <tags>AWS Amazon aws-sdk-v4</tags>
+    <icon>images\AWSLogo.png</icon>
+    <dependencies>
+      <group targetFramework="net472">
+        <dependency id="AWSSDK.Core" version="4.0.1.0-preview" />
+        <dependency id="AWSSDK.CloudFront" version="4.0.1.0-preview" />
+        <dependency id="BouncyCastle.Cryptography" version="2.4.0" />
+      </group>
+      <group targetFramework="netstandard2.0">
+        <dependency id="AWSSDK.Core" version="4.0.1.0-preview" />
+        <dependency id="AWSSDK.CloudFront" version="4.0.1.0-preview" />
+        <dependency id="BouncyCastle.Cryptography" version="2.4.0" />
+      </group>
+    </dependencies>
```

Comment:
You are missing the dependency groups for .NET Core 3.1 and .NET 8.

Reply:
Hmm I tested the nuget package locally and it worked (for net 8), and given that the `CrtIntegration` nuspec file doesn't have a target for netcore31 and net 8, I don't think I need one here, unless the `CrtIntegration` nuspec is doing something different.

Reply:
I wasn't sure, and looks like the service nuspec's do this as well so I added them in the latest revision

Reply:
I assumed it worked by resolving to use the .NET Standard 2.0 version of the assembly.